js/.husky/pre-commit

```diff
@@ -1,2 +1,3 @@
+#!/bin/sh
```

Comment:
This change seems unrelated, and the docs for husky state that this will always execute with `sh`, so there should be no need to restore this. Let's remove it here.

Reply:
Husky throws an error when this is not here. 

See here: https://github.com/conventional-changelog/standard-version/issues/254#issuecomment-424669323

This is conventional-changelog, but the error is about husky. Without this line, I am unable to commit.

Reply:
Might be because I am on Windows?

Reply:
But you are right. This change does not belong in this PR so I've removed it.